cpp/daal/src/algorithms/dtrees/dtrees_train_data_helper.i

```diff
@@ -107,11 +107,22 @@ struct SplitData
     TImpurityData left;
     algorithmFPType featureValue;
     algorithmFPType impurityDecrease;
+    algorithmFPType nTotalWeight;
+    algorithmFPType nLeftWeight;
```

Comment:
What are reasons to insert those fields in the middle instead of in the end?

Reply:
it was fixed, this fields were added in the end